src/test/java/org/broadinstitute/hellbender/tools/copynumber/utils/MergeAnnotatedRegionsIntegrationTest.java

```diff
@@ -50,4 +51,17 @@ public void basicTest() throws IOException {
         Assert.assertEquals(collection.getRecords().get(4), new AnnotatedInterval(new SimpleInterval("2", 1098, 2305),
                 ImmutableSortedMap.of("Num_Probes", "200", "Segment_Mean", "-0.10", "Segment_Call", "0")));
     }
+
+    @Test(expectedExceptions = CommandLineException.class)
+    public void requiresReferenceTest() throws IOException {
+        // This test is a bit more like the real world
+        final File outputFile = File.createTempFile("mergeannotatedregions", ".seg");
+        final List<String> arguments = new ArrayList<>();
+        arguments.add("--" + CopyNumberStandardArgument.SEGMENTS_FILE_LONG_NAME);
+        arguments.add(SIMPLE_TEST_FILE);
+        arguments.add("-" + StandardArgumentDefinitions.OUTPUT_SHORT_NAME);
+        arguments.add(outputFile.getAbsolutePath());
```

Comment:
In the future consider using `ArgumentsBuilder` to build up your integration test command lines (though the manual method here is fine too)